api/NICE.NextWeb.API/CacheManager/NiceOcelotCacheManagerCache.cs

```diff
@@ -21,7 +21,10 @@ public NiceOcelotCacheManagerCache(ICacheManager<T> cacheManager, IHttpContextAc
 
         public void Add(string key, T value, TimeSpan ttl, string region)
         {
-            _cacheManager.Put(new CacheItem<T>(key, region, value, ExpirationMode.Absolute, ttl));
+            if (value is CachedResponse { StatusCode: System.Net.HttpStatusCode.OK } responseVal)
```

Comment:
You made it even better using the property pattern, very nice indeed. Much better than the original, good work